src/event-stream/event-server.ts

```diff
@@ -728,12 +729,15 @@ export async function startEventServer(opts: {
       .json({ status: 'ready', msg: 'API event server listening for core-node POST messages' });
   });
 
+  const handleRawEventRequest = asyncHandler(async req => {
+    await messageHandler.handleRawEventRequest(req.path, req.body, db);
+  });
+
   app.post(
     '*',
```

Comment:
I think this handler can be removed completely now since event write is being moved to each event handler below 👍